packages/opentelemetry-basic-tracer/src/BasicTracer.ts

```diff
@@ -34,7 +34,7 @@ export class BasicTracer implements types.Tracer {
   private readonly _binaryFormat: types.BinaryFormat;
   private readonly _httpTextFormat: types.HttpTextFormat;
   private readonly _sampler: types.Sampler;
-  private readonly _scopeManager: ScopeManager;
+  protected readonly _scopeManager: ScopeManager;
```

Comment:
see comment on NodeTracer

Reply:
ok, I make it public in order to do `plugin._tracer.scopeManager.bind(request)`

Reply:
@vmarchaud see discussion here https://github.com/open-telemetry/opentelemetry-js/pull/161#discussion_r310792407